test/local/account_routes.js

```diff
@@ -849,6 +869,9 @@ test('/account/login', function (t) {
       config.signinConfirmation.sample_rate = 1
 
       return runTest(route, mockRequest, function (response) {
+        t.equal(mockDB.createSessionToken.callCount, 1, 'db.createSessionToken was called')
+        var tokenData = mockDB.createSessionToken.getCall(0).args[0]
+        t.ok(tokenData.tokenVerificationId, 'sessionToken was created unverified')
```

Comment:
`mustVerify` is never returned to the user, only `verified`, `verificationReason` and `verificationMethod`